app/models/discourse_post_event/event.rb

```diff
@@ -53,12 +53,14 @@ def set_next_date
         end
       end
 
-      publish_update!
-      invitees.update_all(status: nil, notified: false)
+      invitees.where.not(status: Invitee.statuses[:going]).update_all(status: nil, notified: false)
+
       if !next_dates[:rescheduled]
         notify_invitees!
         notify_missing_invitees!
       end
+
+      publish_update!
```

Comment:
Not really related, but I think we should publish the update once we are done, and not randomly in the middle of the sequence